toolkits/global/packages/global-ratings-survey/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@springernature/global-ratings-survey",
+  "version": "0.0.1",
+  "license": "MIT",
+  "description": "html form based component for gathering user rating data",
+  "keywords": [],
+  "author": "Springer Nature",
+  "brandContext": "^29.0.3",
+  "dependencies": {
+    "@springernature/global-forms": "^5.0.0"
```

Comment:
I see this conversation has already been marked as resolved, but one of the underlying principles of the design system, as it is now, is that components CANNOT depend on other components. If/when we move to a single release then inter-component dependencies is not so much of an issue and we can rethink the guidance.

It is almost inevitable in this scenario that you will end up with the version of context defined for this component and the version defined for (this version of) global-forms becoming out of sync.

> there is a possibility that at any time this component effectively has a dependency on two different versions of Brand Context at the same time

This is not the case,  we are only ever dependent on _the version defined in this file_ (`29.0.3`) - even by using that version of global forms the context that is installed with this component will be the one used. We just have to hope that forms works with the one used. This is how it works for the demon, for a consumer you are always reliant on a compatible version of context being installed directly from their dependencies (which I am not always sure is the case, but should be an issue solved by the single version)

In summary; I don't see an option of doing anything other than what you have done in order for this to work properly, but at the moment it is most definitely an anti-pattern and not something we want done anywhere else. As such I think (1) we need to get the single version work done as a priority, (2) this component should never be any more than experimental (`0.0.x`) until that is done, and (3) We should provide a very clear on the README that this is temporary until the above happens, and is NOT a pattern to be followed at this time.

Reply:
I have added warnings to the README for points (2) and (3). Both of those warnings can be removed after (1) is done, as discussed.